src/test/scala/org/broadinstitute/hail/utils/OrderedRDDSuite.scala

```diff
@@ -75,15 +75,15 @@ class OrderedRDDSuite extends SparkSuite {
     }
 
     property("randomlyOrdered") = Prop.forAll(random) { case (nPar, s) =>
-      check(sc.parallelize(s, nPar).toOrderedRDD[Locus])
+      check(sc.parallelize(s, nPar).toOrderedRDD)
     }
 
     property("locusSorted") = Prop.forAll(locusSorted) { case (nPar, s) =>
-      check(sc.parallelize(s, nPar).toOrderedRDD[Locus])
+      check(sc.parallelize(s, nPar).toOrderedRDD)
     }
 
     property("fullySorted") = Prop.forAll(sorted) { case (nPar, s) =>
-      check(sc.parallelize(s, nPar).toOrderedRDD[Locus])
+      check(sc.parallelize(s, nPar).toOrderedRDD)
```

Comment:
Should you call coerce and verify the status for `sorted` and `locusSorted`?  Might also have an empty multi-partition one.